tools/github_stats.py

```diff
@@ -152,7 +152,7 @@ def report(issues, show_urls=False):
     milestone = opts.milestone
     project = opts.project
 
-    print("fetching GitHub stats since %s (tag: %s, milestone: %s)" % (since, tag, milestone), file=sys.stderr)
+    print(f"fetching GitHub stats since {since} (tag: {tag}, milestone: {milestone})", file=sys.stderr)
```

Comment:
Preferably don't change this file due to the refactor in #25023.

Reply:
No problem at all. I reverted the changes to this entire file (including the line above).